packages/cf-component-checkbox/src/Checkbox.js

```diff
@@ -21,7 +21,7 @@ class Checkbox extends React.Component {
           name={this.props.name}
           value={this.props.value}
           checked={this.props.checked}
-          onChange={this.handleChange}/>
+          onChange={this.handleChange.bind(this)}/>
```

Comment:
The react docs recommend doing binds once in the constructor instead of every render. 

Although I don't actually know of any cases where performance is actually affected.. it probably doesn't hurt to make that optimization